apps/extension/src/api/modalAxios.ts

```
@@ -0,0 +1,9 @@
+import apiRequest from './axiosInstance';
+
+export const getRemindTime = async (time: String) => {
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)